src/index.html

```diff
@@ -7,6 +7,276 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="header-logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__list-link is-active"> apple </a>
```

Comment:
```suggestion
            <a href="#" class="nav__list-link is-active">apple</a>
```
Зачем тут эти пробелы?